### PR TITLE
bindgen - tidying up benesch's bindgen PR

### PR DESCRIPTION
**Pull request by rrichardson**
* merged up to master
* Fixed up travis for bindgen (clang dependencies) 
* hid max_align_t from bindgen due to (this bug) [https://github.com/rust-lang-nursery/rust-bindgen/issues/550]
* tweaked tests to pass
* Oh almost forgot.  I got tired of rustfmt adding a trailing comma and breaking try_ffi, so I tweaked try_ffi to require a trailing command and adjusted the rustfmt.toml accordingly (there are more vertical function calls than not, so it's better this way ;) )

### CODE REVIEW

**Comment by rrichardson** [comment]
@spacejam - could you be so kind as to review and merge?   Thanks.

File: .travis.yml [hunk 1]
```diff
@@ -10,8 +10,12 @@ addons:
   apt:
     sources:
       - ubuntu-toolchain-r-test
+      - llvm-toolchain-trusty
     packages:
       - g++-5
+      - llvm-3.9-dev
+      - libclang-3.9-dev
+      - clang-3.9
```
Comment (spacejam) [inline .travis.yml]: 👍

**Comment by spacejam** [comment]
Sorry for the delay, I've been moving continents and have been underwater with logistics and visa stuff!

This is awesome! Nice work!

**Pull request merged**